hype-submitter/src/test/java/com/spotify/hype/runner/KubernetesDockerRunnerTest.java

```diff
@@ -76,17 +76,8 @@ public void setUp() throws Exception {
   }
 
   @Test
-  public void setsImage() throws Exception {
-    RunEnvironment env = environment("busybox:1");
-    Pod pod = createPod(env);
-
-    Container container = findHypeRunContainer(pod);
-    assertThat(container.getImage(), is("busybox:1"));
-  }
```

Comment:
I mean that it should test that the image passed to `createPod` is actually put into the pod spec. Also that if the yaml file already contains an image, it should throw.